### PR TITLE
Update bodymovin to the new lottie-web repository with v5.0.0 and filename change

### PR DESCRIPTION
**Pull request by DominikTo**
There is an request (#12687) as well as a PR to add lottie-web as a new lib (#12815), but since this library already exists on cdnjs (with it's old name bodymovin), maybe the right way to move forward is to rename/update the existing library (that stopped autoupdating once it was renamed in v5.0.0, see [this comment](
https://github.com/cdnjs/cdnjs/pull/12815#issuecomment-410786255)), instead of adding a new one.

Closing initial PR (#12904) and resubmitting, to make CI happy.

/cc @bodymovin

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@sashberd you are correct, so we usually just update auto-update config without changing the name on CDNJS.

**Comment by DominikTo** [comment]
Not sure why CI is still complaining about `/ajax/libs/lottie-web/` after undoing the renaming (bodymovin > lottie-web) - that directory doesn't exist.

/cc @PeterDaveHello

**Comment by PeterDaveHello** [comment]
@DominikTo because it's still in the diff/change, the commits need to be squashed into a single one to avoid that ;)

**Comment by DominikTo** [comment]
@sashberd The Problem is, that under the old lib name only versions up to 4.1.3 are available, so everybody who embedded the library via CDNJS under it's old name, won't see newer versions.

**Comment by sashberd** [comment]
@DominikTo You said, what I said, but in another words. Lets finish  #12815 and leave this old lib as is. Then old and new lib will supported

**Comment by DominikTo** [comment]
I'd still propose having "one lib" (there is no old lib and new lib; it just got renamed) that has _all_ version history, instead of one under the old name up to v4.1.3 and then a new one starting at v5.0.0 (that's also what @PeterDaveHello seemed to suggest), but: Whatever gets the latest version of this lib on CDNJS.

Feel free to close this PR, if this is not the way forward.

**Comment by DominikTo** [comment]
B.t.w.: No idea why jsonlint is complaining here: 
https://ci.cdnjs.com/cdnjs/cdnjs/22558

File: ajax/libs/bodymovin/package.json [hunk 1]
```diff
@@ -15,13 +15,13 @@
     "export"
   ],
   "license": "MIT",
-  "homepage": "https://github.com/bodymovin/bodymovin/wiki",
+  "homepage": "https://github.com/airbnb/lottie-web/blob/master/LICENSE.md",
```
Comment (sashberd) [inline ajax/libs/bodymovin/package.json]: The homepage should be https://airbnb.io/lottie/

Comment (sashberd) [inline ajax/libs/bodymovin/package.json]: The homepage should be https://airbnb.io/lottie/

Comment (DominikTo) [reply ajax/libs/bodymovin/package.json]: Updated.

**Comment by DominikTo** [comment]
Thanks for your feedback @PeterDaveHello.

I'd be totally fine with just getting this merged, to get versions >=5.0.0 under the existing/old lib name on CDNJS.

Maybe for the future (not related to this issue) there should be some general approach to rename libs (and potentially forward/alias from old name).

**Comment by bodymovin** [comment]
Hi, lottie dev here.
I'm fine too using the old player name.
If people get confused we can revisit this decision in the future.

**Comment by PeterDaveHello** [comment]
@bodymovin thank you so much

**Comment by sashberd** [comment]
@PeterDaveHello Thus #12815 has to be closed?

**Comment by DominikTo** [comment]
Unless there are any remaining issues, It would be great to merge this, before @PeterBot complains that the branch is too old and has to be rebased. /cc @sashberd, @PeterDaveHello.

**Comment by DominikTo** [comment]
Thanks, @sashberd!

**Comment by DominikTo** [comment]
Should this one also get tagged as in progress?
/cc @sashberd, @PeterDaveHello

**Comment by bodymovin** [comment]
Hi, is there anything missing that I can help with?

**Pull request merged**